### ISSUE TITLE
1.4: link IDs spelling should not be checked

### ISSUE THREAD

**Issue by jouvin** [stfc/markdown-spellchecker#11]
When using link IDs to define anchors, the link ID name should not be checked. As an example:

```
[tl_intro]: {% link _template_library/00-introduction.md %}
A specific set of templates is also stored in the plenary templates: [the template library][tl_intro].
```
would give the (unexpected) error:
```
ERROR:  |A specific set of templates is also stored in the plenary templates: [the template library][tl_intro].
```

**Comment by jrha** [comment]
As in `tl_intro` is being checked and should not be?

**Comment by jouvin** [comment]
Yes. Sorry, I didn't pay attention that I skipped an important part of the error message. Here the problem is tjat `tl` is checked (it seems that every word separated by an `_` is checked separately. You can remove the `tl` entry added in the dictionnary in https://github.com/quattor/quattor.github.com/pull/245 to reproduce the problem (with the PR content).

**Comment by jouvin** [comment]
BTW, the same problem applies is you add attributes to the link like `{:data-proofer-ignore=""}`. Everything between curly braces should be ignored.
